README.md

```diff
@@ -0,0 +1,102 @@
+# Madison
+
+[![Build Status](https://api.travis-ci.org/opengovfoundation/madison.svg?branch=master)](https://travis-ci.org/opengovfoundation/madison)
+[![Scrutinizer Code Quality](https://img.shields.io/scrutinizer/g/opengovfoundation/madison.svg)](https://scrutinizer-ci.com/g/opengovfoundation/madison?branch=master)
+
+**This is the branch for Madison 3.0**
```

Comment:
I think the plan is to land this PR to `dev`, so maybe change this to be something like "Madison 3.0 is here <link to a changelog/what's new>"?